src/test/unittest/unittest.sh

```diff
@@ -1402,6 +1403,21 @@ function require_fs_type() {
 	exit 0
 }
 
+
+#
+# require_build_type -- verify if filesystem supports fallocate
+#
+# Must be AFTER setup() because $DIR must exist (if $1 points to a file inside $DIR)
```

Comment:
This function doesn't care about $DIR. It's caller's responsibility to pass correct path.